RecoEgamma/EgammaIsolationAlgos/plugins/EgammaIsoHcalDetIdCollectionProducer.cc

```diff
@@ -17,13 +17,16 @@ EgammaIsoHcalDetIdCollectionProducer::EgammaIsoHcalDetIdCollectionProducer(const
   phosToken_ = 
   	  consumes<reco::PhotonCollection>(iConfig.getParameter< edm::InputTag >("phosLabel"));
 
+  ootPhosToken_ = 
+  	  consumes<reco::PhotonCollection>(iConfig.getParameter< edm::InputTag >("ootPhosLabel"));
```

Comment:
Well, with cloning, we would still need an input electron and PFclusters input, correct?  And the producer unifies all duplicate detIds.  

We would still need to modify whatever producer takes the interesting Hcal/ES detIds to also include this new collection, and then unify it to drop any repeat detIds.